src/bugreporter/odemis_bugreporter.py

```diff
@@ -293,9 +292,6 @@ def compress_files(self):
                     if os.path.isfile(f):
                         logging.debug("Adding file %s", f)
                         archive.write(f, os.path.basename(f))
-                    elif os.path.isdir(f):
```

Comment:
Leave the `isdir()` check, and implement it with something like this:
```
for top, dirs, files in os.walk(f):
    for subf in files:
        full_path = os.path.join(top, subf)
        archive.write(full_path, full_path[len(f)+1:])
```

Reply:
When we use the second argument in archive.write, we prevent the archive from creating subfolders. So instead of var/log/odemis.log in the archive we only have odemis.log directly in the archive directory. Is this desired? I think that especially for the overlay report, it might be nice to have it in a separate folder.